.github/workflows/instrumentation_tests.yml

```diff
@@ -38,42 +38,42 @@ jobs:
-  browserstack-instrumentation-tests:
-    name: Browserstack Instrumentation tests
-    runs-on: ubuntu-latest
-    timeout-minutes: 55
-    steps:
-      - uses: actions/checkout@v2
-      - uses: gradle/wrapper-validation-action@v1
-      - uses: actions/setup-java@v2
-        with:
-          distribution: 'zulu'
-          java-version: '11'
-      - uses: actions/cache@v2
-        with:
-          path: |
-            ~/.gradle/caches
-            ~/.gradle/wrapper
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: |
-            ${{ runner.os }}-gradle-
-
-      - name: 'Build'
-        run: ./gradlew :paymentsheet-example:assembleDebugAndroidTest :paymentsheet-example:assembleDebug
-
-      - name: 'Install pip modules'
-        run: pip install requests_toolbelt requests
-
-      - name: 'Run BrowserStack tests'
-        env:
-          BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
-          BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
-        run: python scripts/browserstack.py --test --apk paymentsheet-example/build/outputs/apk/debug/paymentsheet-example-debug.apk --espresso paymentsheet-example/build/outputs/apk/androidTest/debug/paymentsheet-example-debug-androidTest.apk
-
-      - name: Notify failure endpoint
-        id: notifyFailureEndpoint
-        if: failure()
-        run: ./scripts/notify_failure_enpiont.rb ${{ secrets.SDK_FAILURE_NOTIFICATION_ENDPOINT }} ${{ secrets.SDK_FAILURE_NOTIFICATION_ENDPOINT_HMAC_KEY }} ${{ github.run_id }}
+#  browserstack-instrumentation-tests:
+#    name: Browserstack Instrumentation tests
+#    runs-on: ubuntu-latest
+#    timeout-minutes: 55
+#    steps:
+#      - uses: actions/checkout@v2
+#      - uses: gradle/wrapper-validation-action@v1
+#      - uses: actions/setup-java@v2
+#        with:
+#          distribution: 'zulu'
+#          java-version: '11'
+#      - uses: actions/cache@v2
+#        with:
+#          path: |
+#            ~/.gradle/caches
+#            ~/.gradle/wrapper
+#          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+#          restore-keys: |
+#            ${{ runner.os }}-gradle-
+#
+#      - name: 'Build'
+#        run: ./gradlew :paymentsheet-example:assembleDebugAndroidTest :paymentsheet-example:assembleDebug
+#
+#      - name: 'Install pip modules'
+#        run: pip install requests_toolbelt requests
+#
+#      - name: 'Run BrowserStack tests'
+#        env:
+#          BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+#          BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+#        run: python scripts/browserstack.py --test --apk paymentsheet-example/build/outputs/apk/debug/paymentsheet-example-debug.apk --espresso paymentsheet-example/build/outputs/apk/androidTest/debug/paymentsheet-example-debug-androidTest.apk
+#
+#      - name: Notify failure endpoint
```

Comment:
rather than nuking the whole thing, would it make sense to just comment out the failure endpoint?

Reply:
I don't see why keep running them until they're fixed, but sure